corehq/apps/es/__init__.py

```diff
@@ -64,4 +65,5 @@ def canonical_name_adapter_map():
     }
     if settings.IS_SAAS_ENVIRONMENT:
```

Comment:
FYI, this is changing to a new setting in https://github.com/dimagi/commcare-hq/pull/36612

Reply:
When do you think your PR will be merged (and `ENABLE_BHA_CASE_SEARCH_ADAPTER` set to True on the correct environments)? I'd ideally like to have this ready for tomorrow's deploy.

Reply:
Merged. [`ENABLE_BHA_CASE_SEARCH_ADAPTER` has been deployed on staging and production](https://dimagi.atlassian.net/browse/SAAS-11593?focusedCommentId=385062).